### PR TITLE
Parts/methods test

### PR DESCRIPTION
**Pull request by mengolicious**
https://trello.com/c/8lpOhn4z
{ Link to applicable Invision designs }

### Done Checklist

* [x] Task has been completed as per the Trello card
* [ ] All acceptance criteria are met
* [x] Developer has tested this locally
* [x] Developer has reviewed their own PR
* [ ] Design has been reviewed by design team
* [x] Tests have been added or updated
* [ ] Views are responsive for mobile devices
* [ ] Feature files have been written or updated
* [ ] Changelog updated

### CODE REVIEW

File: meteor-app/imports/api/orders/orders.test.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    })
+
+    it('success on removing an order', () => {
+      const order = Orders.findOne(orderId)
+      expect(() => Meteor.call('orders.remove', order)).to.not.throw()
+    })
+
+    it('success on updating an order', () => {
+      const orderedParts = goodOrder.orderedParts
+      expect(() => Meteor.call('orders.update', orderId, orderedParts)).to.not.throw()
```
Comment (mikkelking) [inline meteor-app/imports/api/orders/orders.test.js]: This no longer matches the parameters in orders.update (as per the other PR). It doesn't make sense to not put this file in the same PR - the tests should go with the source code they are testing

Comment (mengolicious) [reply meteor-app/imports/api/orders/orders.test.js]: Realized this after the other PR was submitted, have discussed this with the team and will be making some changes to the tests.
Is it okay to keep this PR open as the other PR has already been merged

Comment (mikkelking) [reply meteor-app/imports/api/orders/orders.test.js]: Please fix and push your changes and I can approve

**Comment by mikkelking** [comment]
Good work Meng!

**Pull request merged**